Jellyfin.Plugin.Reports/Jellyfin.Plugin.Reports.csproj

```
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+<PropertyGroup>
+  <TargetFrameworks>netstandard2.0;</TargetFrameworks>
```

```suggestion
  <TargetFramework>netstandard2.0</TargetFramework>
```
This and indentation are the last nitpicks, I swear